### PR TITLE
[feat] [Styles] Responsiveness

### PR DESCRIPTION
**Pull request by shivampandey0**
- Error Route for Wildcard URLs
- useMediaQuery hook to get screen size changes in JSX
-  CSS class refactors for responsiveness

### CODE REVIEW

**Review by Ronak-Mutha — APPROVED** [review]

**Review by GuptaEesh — COMMENTED** [review]
Good that you added the error route.
Good work on responsiveness.
Good to merge

File: src/hooks/useMediaQuery.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+const useMediaQuery = (query) => {
+  const [matches, setMatches] = useState(false);
+
+  useEffect(() => {
+    const media = window.matchMedia(query);
+    if (media.matches != matches) {
+      setMatches(media.matches);
+    }
+    console.log(query);
```
Comment (Ronak-Mutha) [inline src/hooks/useMediaQuery.jsx]: Remove console.logs before merging

File: src/pages/pomodoro/Pomodoro.jsx [hunk 2]
```diff
@@ -16,6 +17,20 @@ export const Pomodoro = () => {
...
+   * Media Queries
+   */
+
+  const isDesktop = useMediaQuery("(min-width:960px)");
+  const isTablet = useMediaQuery("(max-width:600px)");
+
+  const timerSize = isDesktop ? 360 : isTablet ? 200 : 270;
+  const strokeSize = isDesktop ? 24 : isTablet ? 12 : 18;
+
+  // console.log(timerSize);
```
Comment (Ronak-Mutha) [inline src/pages/pomodoro/Pomodoro.jsx]: Remove unnnecessary or non-informative comments.

**Review by Ronak-Mutha — APPROVED** [review]

**Comment by Ronak-Mutha** [comment]
Apart from the mentioned comment about console, good to merge.

**Pull request merged**